### ISSUE TITLE
Type checker hangs indefinitely in v1.5.1

### ISSUE THREAD

**Issue by pelotom** [TypeStrong/fork-ts-checker-webpack-plugin#344]
This was not happening prior to v1.5.1.

Steps to reproduce:

```
npx create-react-app bug
cd bug
yarn eject
```

(type `y` to confirm)

```
yarn add typescript
yarn tsc --init
yarn add fork-ts-checker-webpack-plugin@1.5.1
```

Edit `webpack.config.js`:

```diff
-const ForkTsCheckerWebpackPlugin = require('react-dev-utils/ForkTsCheckerWebpackPlugin');
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
```

Run `yarn start`, and it hangs forever on:

```
Files successfully emitted, waiting for typecheck results...
```

**Comment by johnnyreilly** [comment]
Thanks for reporting @pelotom. This is likely down to this change which moved fork-ts-checker-webpack-plugin to log everything to stderr

https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/issues/321

@akx @piotr-oles it looks like logging to stderr might be problematic. Please see the details above. Is there anything we can do to remedy this? If not perhaps we should consider reverting?

**Comment by johnnyreilly** [comment]
@pelotom would you like to raise a PR that reverts:

https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/commit/a89fdaf081b845e82eb8a1b578c597da6e24d2a2

If there are no suggestions of alternate approaches then we should look to merge that and get it released as `1.5.2`

**Comment by pelotom** [comment]
Sure, here ya go: https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/pull/345
